pallets/order-book/src/traits.rs

```diff
@@ -104,25 +104,58 @@ where
         price: &OrderPrice,
     ) -> Option<PriceOrders<T::OrderId, T::MaxLimitOrdersForPrice>>;
 
+    /// Returns whether there is no place for orders inside the bid price
+    fn is_bids_full(
```

Comment:
```suggestion
    fn are_bids_full(
```

Reply:
My logic was that `bids` is a storage value, so this checks if the value is full.
It seems non-intuitive, I agree. What do you think about `is_bid_price_full`?

Reply:
`is_bid_price_full` looks incorrect

Reply:
Ok